projects/igniteui-angular/src/lib/chips/chip.component.ts

```diff
@@ -306,6 +311,18 @@ export class IgxChipComponent implements AfterViewInit {
     @ViewChild('removeBtn', { read: ElementRef })
     public removeBtn: ElementRef;
 
+    /**
+     * @hidden
+     */
+    @ContentChildren(IgxPrefixDirective)
+    public prefixes: QueryList<IgxPrefixDirective>;
```

Comment:
No need of this changes. The required styling will be done in components' theme files.